test/e2e/infrastructure/infrastructure.go

```diff
@@ -0,0 +1,56 @@
+package setup
+
+import (
+	"fmt"
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
+	"k8s.io/utils/pointer"
+)
+
+func ()  {
+
+}
+
+func setup() error {
+	mc := machineconfigv1.MachineConfig{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "nri-enablement",
+			Labels: map[string]string{
+				machineconfigv1.MachineConfigRoleLabelKey: "worker",
+			},
+		},
+		Spec: machineconfigv1.MachineConfigSpec{
+			Config: runtime.RawExtension{},
+		},
+	}
+
+	ignitionConfig := &igntypes.Config{
+		Ignition: igntypes.Ignition{
+			Version: defaultIgnitionVersion,
+		},
+		Storage: igntypes.Storage{
+			Files: []igntypes.File{
+				{
+					Node: igntypes.Node{
+						Path: "/etc/crio/crio.conf.d/99-nri-enablement.conf",
+					},
+					FileEmbedded1: igntypes.FileEmbedded1{
+						Contents: igntypes.Resource{
+							Source: pointer.String(fmt.Sprintf("%s,%s", defaultIgnitionContentSource, contentBase64)),
+						},
+						Mode: pointer.Int(448),
```

Comment:
this looks like a weird permission set, what do we need here?

Reply:
This is a standard unix file mode. 
Let's see if we can remove it and use default

Reply:
Actually I removed it on latest commit. I should layout the commit history in more organized way, sorry for that.

Reply:
but in standard unix octal `448` makes not much sense. Anyway, if you fixed differently it's fine